modules/data-mover-for-csi-snapshots.adoc

```diff
@@ -0,0 +1,197 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="data-mover-for-csi-snapshots_{context}"]
+= Data Mover for CSI snapshots
+
+
+The OADP 1.1 Data Mover enables customers to back up CSI volume snapshots to a remote object store and to restore stateful applications from the store in the event of a failure, accidental deletion or corruption of the cluster.
+The OADP 1.1 Data Mover solution uses the Restic option of VolSync.
+
+[NOTE]
+=====
+Data Mover supports backup and restore of CSI volume snapshots only.
+=====
+
+.Prerequisites
+
+* Have a stateful application running in a separate namespace.
+
+* The `StorageClass` and `VolumeShapshotClass` CRs support CSI.
+
+* Only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
+
+* Only one `storageClass` CR has the annotation `storageclass.kubernetes.io/is-default-class: true`.
+
+* Include the label `{velero-domain}/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` CR to indicate for Velero which to use.
+
+* Install the VolSync Operator using the Operator Lifecycle Manager (OLM). Administrators can link:https://docs.openshift.com/container-platform/4.10/operators/admin/olm-adding-operators-to-cluster.html[install Operators on clusters]. Non-administrators can link:https://docs.openshift.com/container-platform/4.10/operators/user/olm-installing-operators-in-namespace.html[install Operators in namespaces].
++
+[NOTE]
+====
+The VolSync Operator is required only for use with the Technology Preview Data Mover. The Operator is not required for using OADP GA features.
+====
+
+* Install the link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[OADP Operator] using OLM.
+
+* Install the link:https://volsync.readthedocs.io/en/stable/installation/index.html[VolSync operator] using OLM.
```

Comment:
please add downstream link references.
If there is not specific downstream Volsync installation doc, please refer to the general procedure of installing an operator:
https://docs.openshift.com/container-platform/4.11/operators/admin/olm-adding-operators-to-cluster.html

Reply:
@mperetzred  See above. I can only use an external link here. And why not? There are lots of external links in our documentation. And this procedure is exactly what the user needs here.

Reply:
@sbeskin-redhat that's because this reference of how to install volsync is by using help, while we encourage to install it as an operator on the official docs :)

Reply:
@mperetzred Then I will have to move the xref to 'Additional resources' at the end of the section.